### PR TITLE
[VarDumper] Add support for Fiber

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        |

image

### CODE REVIEW

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Component/VarDumper/Caster/FiberCaster.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * Casts Reflector related classes to array representation.
```
Comment (Kocal) [inline src/Symfony/Component/VarDumper/Caster/FiberCaster.php]:
```suggestion
 * Casts Fiber related classes to array representation.
```

Comment (lyrixx) [reply src/Symfony/Component/VarDumper/Caster/FiberCaster.php]: Fixed, thanks

File: src/Symfony/Component/VarDumper/Caster/FiberCaster.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarDumper\Caster;
+
+use Symfony\Component\VarDumper\Cloner\Stub;
+
+/**
+ * Casts FiberCaster related classes to array representation.
```
Comment (Kocal) [inline src/Symfony/Component/VarDumper/Caster/FiberCaster.php]: It should be `Fiber` instead of `FiberCaster` no?

Comment (lyrixx) [reply src/Symfony/Component/VarDumper/Caster/FiberCaster.php]: Rah, I'm stupid. Thanks

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/VarDumper/Caster/FiberCaster.php (hunk 2 above)
Comment (Kocal) [reply src/Symfony/Component/VarDumper/Caster/FiberCaster.php]: (you are not stupid! 😊)

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @lyrixx.

**Pull request merged**